src/main/java/com/example/LoyaltyCard.java

```diff
@@ -0,0 +1,36 @@
+package com.example;
+
+import static com.google.common.base.Preconditions;
+
+
+/**
+ * Each customer has his own card that serves as an additional identifier.
+ */
+public final class LoyaltyCard {
+    /**
+     * Customer's card id.
+     */
+    private final String number;
+
+    /**
+     * Constructor creates a new card with the passed number.
+     *
+     * <p>Checking the card number.
+     *
+     * @param number a unique card number, consisting of digital characters.
```

Comment:
How do we know that the number is unique?

Can we put 100_000 characters into this sequence?

What happens if it's just a couple of characters?